src/constants.ts

```diff
@@ -16,29 +16,43 @@ export const configDir = applicationConfigPath('devcert');
 export const configPath: (...pathSegments: string[]) => string = path.join.bind(path, configDir);
 
 export const domainsDir = configPath('domains');
-export const pathForDomain: (domain: string, ...pathSegments: string[]) => string = path.join.bind(path, domainsDir)
+export const pathForDomain: (domain: string | string[], ...pathSegments: string[]) => string = path.join.bind(path, domainsDir)
```

Comment:
I don't think this can be `string[]` here, since `path.join()` itself only accepts an arg array of `string` types.

In this context ,`path.join.bind(path, domainsDir)` basically becomes 
```js
(domain, ...pathSegments) => path.join(domainsDir, domain, ...pathSegments)
```

This is what I get in a node shell:
```sh
> configDir = '/test'
'/test'
> configPath = path.join.bind(path, configDir)
[Function: bound join]
> domainDir = configPath('domains')
'\\test\\domains'
> pathForDomain = path.join.bind(path, domainDir)
[Function: bound join]
> pathForDomain(['localhost', 'test.localhost'], 'private-key.key')
Thrown:
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:427:7)
```

Reply:
Personally, I think that using `path.join.bind()` like this is kind of an obscure usage, and can lead to confusion.  I know it took me a moment when I first looked at it, and this is the only place I've ever seen it used.

@zetlen how would you feel about changing these to simple wrapper functions, instead of bound `path.join` functions?

Reply:
Oops - I had figured that out while testing, forgot to push - the fixed version is now here.